### ISSUE TITLE
expected operator between binary operands

### ISSUE THREAD

**Issue by acarl005** [decaffeinate/decaffeinate#1113]
This project could be a life-saver for me! I'd like to use it on an old script but unfortunately its crashing.

Decaffeinate is crashing on my CoffeeScript input:

```coffee
_.zip(list, stats, [0...list.length], (exts.length > 0 and exts or [0...list.length]))
```

LINK TO [(repl)](http://decaffeinate-project.org/repl/#?evaluate=true&stage=full&code=_.zip(list%2C%20stats%2C%20%5B0...list.length%5D%2C%20(exts.length%20%3E%200%20and%20exts%20or%20%5B0...list.length%5D)))

I get this error:

```
ls-color.coffee: expected operator between binary operands
  253 | sort = (list, stats, exts) ->
  254 |     (exts ?= []);
> 255 |     l = _.zip(list, stats, [0...list.length], (exts.length > 0 and exts or [0...list.length]))
      |                                                                            ^^^^^^^^^^^^^^^^^
  256 |     if args.kind
  257 |         if exts == [] then return list
```

**Comment by alangpierce** [comment]
Thanks for reporting! I think I should be able to get to this pretty soon.

**Comment by alangpierce** [comment]
Here's a reduced test case:
```coffee
a or [b...c]
```

[(repl)](http://decaffeinate-project.org/repl/#?evaluate=true&stage=full&code=a%20or%20%5Bb...c%5D)

**Comment by alangpierce** [comment]
Will be fixed when #1114 is merged and released, which should be in the next few hours.

**Comment by acarl005** [comment]
It works! Thanks for such a quick turnaround.
